Angular-Debugging/.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "chrome",
+      "request": "launch",
+      "name": "Launch Chrome with ng serve",
+      "url": "http://localhost:4200/#",
+      "webRoot": "${workspaceRoot}",
+      "sourceMaps": true,
+      "trace": true,
+      "userDataDir": "${workspaceRoot}/.vscode/chrome"
```

Comment:
Agreed .. Removed these in PR #5: Remove sourceMaps, trace, userDataDir from launch.json.